content/en/docs/combined-matrix.md

```diff
@@ -0,0 +1,342 @@
+
+The following table lists the authentication protocols that Armory Enterprise supports:
+
+| Identity provider     | Armory Enterprise      | Note                                                                  |
+|-----------------------|------------------------|-----------------------------------------------------------------------|
+| SAML                  | All supported versions |                                                                       |
+| OAuth 2.0/OIDC        | All supported versions | Can use Auth0, Azure, GitHub, Google, Okta, OneLogin, or Oracle Cloud |
+| LDAP/Active Directory | All supported versions |                                                                       |
+| x509                  |                        |                                                                       |
+
+## Authorization
+
+![Generally available](/images/ga.svg) ![OSS](/images/oss.svg) ![Armory](/images/armory.svg)
+
+The following table lists the authorization methods that Armory Enterprise supports: 
+
+| Provider              | Armory Enterprise      | Note                                                                            |
+|-----------------------|------------------------|---------------------------------------------------------------------------------|
+| GitHub Teams          | All supported versions | Roles from GitHub are mapped to the Teams under a specific GitHub organization. |
+| Google Groups         |                        |                                                                                 |
+| LDAP/Active Directory | All supported versions |                                                                                 |
+| OAuth 2.0/OIDC        | All supported versions |                                                                                 |
+| SAML                  | All supported versions |                                                                                 |
+
+## Browsers
+
+Spinnaker's UI (Deck) works with most modern browsers.
+>  COMMENT. REMOVE ME BEFORE PUBLISHING.
+>  The above seems like the most maintainable/most non-controversial statement
+
+## Build systems
+
+![Generally available](/images/ga.svg) ![OSS](/images/oss.svg) ![Armory](/images/armory.svg)
+
+The following table lists the CI systems that Armory Enterprise supports:
+
+| Provider           | Version | Armory Enterprise      | Note                |
+|--------------------|---------|------------------------|---------------------|
+| AWS CodeBuild      | n/a     | 2.19.x or later        |                     |
+| GitHub Actions     | n/a     | All supported versions | Webhook integration |
+| Google Cloud Build |         |                        |                     |
+| Jenkins            |         | All supported versions |                     |
+| Travis             |         | All supported versions |                     |
```

Comment:
@aelath are there specific versions of Jenkins or Travis we support? Or should this just be something like "All actively maintained versions"

Reply:
@brian-armory Good question. I *think* we just support the same versions OSS does. For Jenkins that would be `1.x+` (https://spinnaker.io/setup/ci/jenkins/#prerequisites). Looks like Jenkins LTS minimum version is `1.409.1`, might wanna use that? No versions listed for Travis (https://spinnaker.io/setup/ci/travis/), so I think saying "All actively maintained versions" is fine.